### DISCUSSION TITLE
What is the maximum number of characters in the 'password' field when registering an account on GitHub?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#67862] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello guys,

I would like to know what is the maximum number of characters in the 'password' field when registering an account on GitHub. What is the maximum number of characters in the 'password' field when registering an account on GitHub?

**Context**
I read a report on some blog that I don't remember the name that for greater digital security there should be something around 8 characters for the password field. *Would it be better to add more characters to the password field, in addition to what is standard, in this case increasing 16 characters instead of 8 characters?* *Where can I find privacy information on GitHub? How can I make my account more secure and private? What are GitHub's privacy and security guidelines? Is increasing the number of characters in the password field more secure?*

**Comment by SenaJp** [comment, marked as answer]
In particular, I use a paid password manager (1Password). But you can find several free ones like bitWarden for example. I also use two-factor authentication using authy. As an option, you can authenticate through the GitHub app itself, I used that for a while too.

In the github article you will find recommendations for creating stronger and more secure passwords:
https://docs.github.com/pt/authentication/keeping-your-account-and-data-secure/creating-a-strong-password

in the link below you will see how to configure two-factor authentication and what options are available:
https://docs.github.com/pt/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication

**Reply by BrianH12345** [reply]
This doesn't answer the question. The question was what: how long can a GitHub password be?

**Comment by fulltimefoolery** [comment]
The answer is: 72

I know this is an old thread, but it's the first thing that came up after doing a search. So this is for future seekers of the magic number.  

Would be nice if Github would just say so instead of giving this error as I try to dumb down my password hitting backspace until I find the psudo-secret max character limit.

<img width="1188" height="652" alt="2025-07-25_11-52-github-pwtoolong" src="https://github.com/user-attachments/assets/00bd9589-863f-4d26-a388-4d93d8d8c4a7" />
